### PR TITLE
SC-9192 Fix path mapping

### PR DESCRIPTION
**Pull request by uidp**
# Short Description
Access feathers and NestJS apps by their verision prefixes.

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/plugins/axios.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ const unrecoverableErrorCodes = [401, 404, 500];
 
 export default function ({ $axios, store, error }) {
 	$axios.onRequest((config) => {
+		// prepend default '/v1' version prefix to all urls that don't begin with '/v<x>/'
+		if (!config.url.match(/^\/v\d\//)) {
```
Comment (tofixx) [inline src/plugins/axios.js:6]: is this applied only for calls against our api server or globally? i think it should not be applied to other external calls...?!

Comment (uidp) [reply src/plugins/axios.js:6]: unfortunately this is globally applied.

**Review by tofixx — DISMISSED** [review]

**Review by pkleybolte — APPROVED** [review]

**Review by tofixx — APPROVED** [review]

**Review by virgilchiriac — DISMISSED** [review]

File: src/components/organisms/AddContentButton.vue [hunk 2]
```diff
@@ -141,7 +141,7 @@ export default {
 					client: element.client,
 					url: element.merlinReference
 						? await this.$axios.$get(
-								`/edu-sharing/merlinToken/?merlinReference=${element.merlinReference}`
+								`/v1/edu-sharing/merlinToken/?merlinReference=${element.merlinReference}`
```
Comment (mamutmk5) [inline src/components/organisms/AddContentButton.vue:144]: It is possible to solve the _v1/_ part without a breaking change?

**Review by mamutmk5 — CHANGES_REQUESTED** [review]
It is possible to solve the _v1_ part without a breaking change?

**Review by mamutmk5 — APPROVED** [review]

**Pull request merged**